components/user_list_row_with_error.jsx

```diff
@@ -0,0 +1,143 @@
+        actions: PropTypes.arrayOf(PropTypes.func),
+        actionProps: PropTypes.object,
+        actionUserProps: PropTypes.object,
+        userCount: PropTypes.number
+    };
+
+    static defaultProps = {
+        extraInfo: [],
+        actions: [],
+        actionProps: {},
+        actionUserProps: {}
+    };
+
+    constructor(props) {
+        super(props);
+        this.state = {};
+
+        this.onError = this.onError.bind(this);
+    }
+
+    onError(errorObj) {
+        this.setState({
+            error: errorObj
+        });
+    }
+
+    render() {
+        let buttons = null;
+        if (this.props.actions) {
+            buttons = this.props.actions.map((Action, index) => {
+                return (
+                    <Action
+                        key={index.toString()}
+                        user={this.props.user}
+                        {...this.props.actionProps}
+                        {...this.props.actionUserProps}
+                        onError={this.onError}
+                    />
+                );
+            });
+        }
+
+        // QUICK HACK, NEEDS A PROP FOR TOGGLING STATUS
+        let email = this.props.user.email;
+        let emailStyle = 'more-modal__description';
+        let status;
+        if (this.props.extraInfo && this.props.extraInfo.length > 0) {
+            email = (
+                <FormattedHTMLMessage
+                    id='admin.user_item.emailTitle'
+                    defaultMessage='<strong>Email:</strong> {email}'
+                    values={{
+                        email: this.props.user.email
+                    }}
+                />
+            );
+            emailStyle = '';
+        } else if (this.props.user.status) {
+            status = this.props.user.status;
+        } else {
+            status = UserStore.getStatus(this.props.user.id);
+        }
+
+        let userCountID = null;
+        let userCountEmail = null;
+        if (this.props.userCount >= 0) {
+            userCountID = Utils.createSafeId('userListRowName' + this.props.userCount);
+            userCountEmail = Utils.createSafeId('userListRowEmail' + this.props.userCount);
+        }
+
+        let error = null;
+        if (this.state.error) {
+            error = (
+                <div className='has-error'>
+                    <label className='has-error control-label'>{this.state.error.message}</label>
+                </div>
+            );
+        }
+
+        return (
+            <div
+                key={this.props.user.id}
+                className='more-modal__row'
+            >
+                <ProfilePicture
+                    src={Client4.getProfilePictureUrl(this.props.user.id, this.props.user.last_picture_update)}
+                    status={status}
+                    width='32'
+                    height='32'
+                />
+                <div className='more-modal__right'>
+                    <div className='more-modal__top'>
```

Comment:
If you did want to reduce some of the duplication here, you could break parts of this out into `UserListRowDetails` and `UserListRowActions` components to use both places